qa/rpc-tests/listtransactions.py

```diff
@@ -20,12 +20,12 @@ def txFromHex(hexstring):
 class ListTransactionsTest(BitcoinTestFramework):
 
     def setup_nodes(self):
-        #This test requires mocktime
-        enable_mocktime()
```

Comment:
did you delete the cache directory?  mocktime (and this test) should work since I changed the mock time to be post fork.  But you need to delete your node cache to force it to be regenerated with the new mocktime.